cmd/configuration.go

```diff
@@ -46,6 +46,7 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 	defaultDocker.SwarmMode = false
+	defaultDocker.SwarmRefreshPeriod = 15
```

Comment:
Sorry, but I made a mistake, I would say `SwarmModeRefreshPeriod` in the way to be homogeneouse with the option `SwarmMode` and avoid misunderstood with the old Swarm orchestrator...